net/instaweb/rewriter/in_place_rewrite_context.cc

```diff
@@ -454,6 +461,16 @@ void InPlaceRewriteContext::RemoveRedundantRelCanonicalHeader(
                   ResponseHeaders::RelCanonicalHeaderValue(url_));
 }
 
+bool InPlaceRewriteContext::IsLoadFromFileBased() {
```

Comment:
@jmarantz add this, I anticipate we may need it later for landing https://github.com/pagespeed/mod_pagespeed/pull/1492